### DISCUSSION TITLE
Gedichte & Gemälde von Hermann Hesse

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#110] [General]
![image](https://user-images.githubusercontent.com/78131082/213596656-54659b0e-c6e0-49cd-93e3-2fe25386c452.jpeg)

![image](https://user-images.githubusercontent.com/78131082/213596661-22465989-2c31-4e5e-8266-4c6380142a6f.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213597191-8a7894df-bdd3-44cf-90e7-0fbb8ba66a84.jpeg)

**Reply by ThomasG777** [reply]
FRÜEHLIG

in dämmrige Grüft
träumt ich lang
vo dine bäum und blaue lüft
vo dim Duft und Vogelgsang

nu liegsch du erschlosse
in Gleis und Zier
Vo Licht übergosse
Wie es Wunder vor mir

Du erkensch mich wieder 
Du locksch mich zart
Es zittert durch all mini Glieder
Dini seligi Gegewart



TRAUM


Es isch immer de glichi Traum
En rotblühende Kastaniebaum
En Garte voll vo Summerflor
Einsam es alts Hus davor

Det, wo de stilli Garte liegt
Hät mini Muetter mich gwiegt
Viellicht- es isch scho lang her-
Staht Garte, Hus und Baum nüme mehr

Viellicht gaht jetzt en Wieseweg
Und Pflug und Egge drüber weg
Vo Heimat, Garte Hus und Baum
Isch nüt bliebe als min Traum






MÄNGISCH


Mängisch wenn en Vogel rueft
Oder en Wind gaht i de Zwige
Oder en Hund bellt im fernste Ghöfft
Denn muess ich lang lusche und schwiege

Mini Seel flieht zrug
Bis vo vor tusig vergessene Jahre
Der Vogel und der wehendi Wind
mir ähnlich und mini Brüeder ware 

Mini Seel wird Baum
Und es Tier und es Wolkewebe
Verwandelt und fremd kehrt sie zrug
Und frögt mich. Wie soll ich Antwort geh?

**Comment by esthicodes** [comment]
Er ging im Dunkel
Er ging im Dunkel gern, wo schwarzer Bäume
Gehäufte Schatten kühlten seine Träume.
Und dennoch litt in seiner Brust gefangen
Nach Licht; nach Licht! ein brennendes Verlangen.
Er wußte nicht, daß über ihm die klaren
Himmel voll reiner Silbersterne waren.
어둠 속을 그는 걸었네
Er ging im Dunkel 1901
어둠 속을 그는 즐겨 걸었네
짙은 수목들 그 아롱진 그늘이 그의 꿈들을 다독이던 곳.
하지만 그의 가슴속에선, 빛에!
빛에 사로잡혀 불타는 갈망이 괴로워했네.
그는 몰랐네. 자기 위에 저리 청명한
순수의 은별 가득한 하늘이 있었다는 걸.
043
042

**Reply by ThomasG777** [reply]
ER GING IM DUNKEL  

Er isch im Dunkel gange gern wo schwarzi Bäum
Ghüfti Schatte Chüelte sini Träum
Und trozdem lidet i sini Brust gfange
Nach Liecht; nach Liecht e brennends Verlange
Er hät nöd gwüsst, dass über ihm die klare
Himmel voll vo reine Silbersterne gsi sind.

**Comment by esthicodes** [comment]
Die leise Wolke
Eine schmale, weiße
Eine sanfte, leise
Wolke weht im Blauen hin.
Senke deinen Blick und fühle
Selig sie mit weißer Kühle
Dir durch blaue Träume ziehen.
044
조용한 구름
Die leise Wolke 1901
가늘고 하얀
부드럽고 조용한
구름 한 점이 파란 하늘에 흘러간다.
눈을 그윽이 뜨고 느껴보렴
멋지게 하얀 시원함으로
네 푸른 꿈들을 질러 떠가는 것을.
045

**Reply by ThomasG777** [reply]
DIE LISLIGI WULCHE

E  schmali, wisi
E sanfti, lisligi
Wulche weht im blaue hi
Senk din Blick und fühl
Selig sie mit wiser chüeli
Dir durch blaui Träum zieht

**Comment by ThomasG777** [comment]
WIE E WELLE

Wie e Welle, die vom Schum gekränzt 
Us blauer Fluet sich voll Verlange reckt
Und müed und schön im grosse Meer verglänzt

Wie e Wulche, die im lislige Wind
Hisegelnd allne Pilger sehnsucht weckt
Und blass und silbrig i de Tag verinnt-

Und wie es Lied am heisse Strasserand
Fremdtönig klingt mit wunderlichem Reim
Und dir sHerz entführt wit über Land-

So weht mis Lebe flüchtig durch Zit,
Isch bald vertönt und mündet doch gheim
Is Rich vo de Sehnsucht und de Ewigkeit

**Comment by ThomasG777** [comment]
Wer mini Fründe sind?-
Zugvögel überm Ozean verirrt,
Schiffbrüchigi Schiffer, Herde ohni Hirt
d Nacht , de Traum, de Heimatlosi Wind

Am Weg lieget hinter mir
Zerstörti Tempel, Liebesgärte
Verwildernd, schwül und sommerzier
Und Fraue mit welken Liebesgebärde,
Und Meer, die ich überfahre han
 
Sie lieged stumm und ohni Spur;
Kennt keine was versunke liegt,
Die Königskrone, die Herrscherstunde,
Die Fründesstirne epheuumwunde

Sie lieget vo mine Lieder gwiegt
Und dämmeret blass i mini Nächt,
Wenn hastig mini schmali Rechti
Mit raschem Stift i mim Lebe wüehlt

Ich ha nie es Ziel errunge,
Mini Fuscht het nie en Feind zwunge
Mis Herz het nie es volls Glück gfühlt -_-

**Comment by ThomasG777** [comment]
Verlorene Klang

Einmal in Chindertage
Bin ich die Wiese lang
Isch cho still treit 
Im Morgewind en Gsang

En Ton in blaue Luft,
Oder en Duft, en bluemige Duft,
Süess het er Duftet, de klang
E ewigkeit lang
Mini ganz Chindheit lang.

Es isch mer nüm bewusst gsi
Erscht jetzt i dene Tage
Hör ich inne i de Brust
Ihn wieder verborge schlage
Und jetzt isch alle Welt mir einerlei,
Will nöd mit de Glückliche tusche,

Will nur luhsche 
Luhsche und stillstah
Wie die duftende Tön göhnd
Und ob es no de Klang vo damals isch

**Comment by ThomasG777** [comment]
Im vierte Chriegsjahr

Wenn au de Abig chalt und trurig isch
und de Rege ruscht
Ich singe doch mis Lied i dere Frischt
Weis nöd , wer luhscht.

Wenn au die Welt in Chrieg und Angscht erstickt,
A manche Ort
Brennt heimlich doch, auch wenn niemed sie erblickt
dLiebi witer

**Comment by ThomasG777** [comment]
Voll Blüete

Voll Blüete staht de Pfirsichbaum
nöd jedi wird zur Frucht
Sie schimmeret hell wie Roseschum
Durch Blau und Wolkeflucht.

Wie Blüete göhnd Gedanke uf
Hundert a jedem Tag-
Lass blüeh! Lass dem Ding de Lauf!
Frög nöd nach em Ertrag!

Es muess au Spiel und Tanze si
Und Blüeteüberfluss,
Suscht wär die welt üs viel zu chli
Und Lebe kein Gnuss

**Comment by ThomasG777** [comment]
Magie vo de Farbe

Gottes Atem hi und wieder,
Himmel obe, Himmel une
Liecht singt tusigfachi Lieder,
Gott wird Welt im farbig bunte,

Wiss zu Schwarz, und Warm zu Chüehl
Fühlt sich immer neu zoge,
Ewig us chaotischem Wüehle,
Klärt sich neu de Regeboge.

So durch üseri Seel wandelt
Tusigfach in Qual und Wonne
Gottes Liecht, erschafft und handelt,
Und mir prieset Ihn als Sunne.

**Comment by ThomasG777** [comment]
Dorfabig

De Schäfer mit de Schaf
Zieht durch die stille Gasse i,
d Hüser wennd schlafe
Und dämmered scho und nicked i

Ich bin i dene Mure
De einzig fremdi Ma zur Stund,
Es trinkt mis Herz mit Truer
De Chelch vo de Sehnsucht bis zum Grund.

Wohi de Weg mich führt,
het überall en Herd brennt;
Nur ich ha nie gspürt,
Was Heimat isch und Vaterland

**Comment by ThomasG777** [comment]
Ich bin en Stern

Ich bin en Stern am Firmament
wo die Welt betrachtet, die Welt verachtet,
Und i de eigene Gluet verbrennt.

Ich bi das Meer, das i de Nacht stürmt,
Das klagendi Meer, das opferschwer 
Zu alte Sünde sich neu türmt.

Ich bi vo eure Welt verbannt
Vom Stolz erzoge, vom Stolz beloge,
Ich bi de König ohni Land.

Ich bi die stummi Lideschaft,
Im Hus ohni Herd, im Chrieg ohni Schwert,
Und chrank a minere eigene Chraft.

**Comment by ThomasG777** [comment]
a en Fründ mit em Gedichtbuech

Was mich je bewegt und erfreut het
Sit de sagehafte Jugentag
All das flüchtige und bunt Zerstreute
A Besinnig und Träumereie,
A Gebet , Werbige und Klage
Findsch uf dene Site wieder.
Ob sie erwünscht oder unnütz sind
wem mir nöd allzu ernschtlich frage?
Nimm sie fründlich uf, die alte Lieder!

Üs de Altgewordene isch das wile
Im Vergangene erlaubt und tröschtlich,
Hinter dene viele tusig Zile
Blüeht es Lebe, und es isch mal Köstlich gsi.
Werde mir zur Recheschaft zoge,
Das mir üs mit sötigem Tand befasst hend,

Traged mir wohl liehchter unseri Laschte
Als die Flüger, die hüt nacht gfloge sind
Als Heeres arme, bluetigi Herde
Als die Herre und Grosse vo dere Erde.

**Comment by ThomasG777** [comment]
Ich ha gloge

ich ha gloge! Ich ha Gloge! Ich bi alt,
Ich bin nöd satt vom Lebe,
Mir macht jedi schön Frauegstalt
No puls und Gedanke erbebe.

Mir träumt no vo Wiber heiss und nackt,
Vo guete und vo schlechte,
Vo wildem Walzer brilliantem Takt
Und vo verliebte Nächt.

Mir träumt vo de Liebi sogar,
vonere schwiegsame und reine,
wie jeni erschti , heilig isch gsi
Und ich chan no um sie weine/hüle

**Comment by ThomasG777** [comment]
Ich Liebe Fraue

Ich liebe Fraue, die wo vo tusig Jahre
Gliebt und vo Dichter Besunge worde sind

Ich liebe Städt, dere ihri leere Mure
Königsgschlechter us alte zite betruret.

Ich liebe Städt, wo entstah werdet,
wenn niemed mehr vo hüt lebt uf Erde.

Ich liebe Fraue - schlanki, wunderbari,
wo ungehorsam rueht im Schoss vo de Jahr.

Sie werdet einmal mit ihre sternebleiche 
Schönheit de Schönheit miner Träum gliche.

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842766-e1640ed3-b162-4fbe-98b2-82cd7cee0909.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842775-e159ed6a-31b9-4292-a4a5-0f7463c139f0.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842794-7e983fb6-9b92-49aa-8708-62341cd718d4.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842801-3168f0ae-729c-4c3f-ba5e-a2ed69d33bbd.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842810-e517c546-7eba-4bf1-9d8c-6097914c40d6.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842818-2ff252ba-85a0-4e4e-9fcb-7c5246e0150a.jpeg)

**Reply by ThomasG777** [reply]
Gebet

Wenn ich einmal vor dim Alitz stah
Denn denk, wie du mich allei gla,
Und denke, wie ich irrend i de Gasse
verwaist und trostlos  gsi i mine weh

Denn denk i jene schreckliche dunkle Nächt
wo ich in not und heissem Heimweh bangt ha
Und wie es chind nach diner hand verlangt ha
Und du mir verseit hesch dini Rechti.

Un denke a jene Zit wo ich als Bueb
Zu dir zrug a jedem Tag cho bin
Und mini Muetter , die mir sbete glehrt het
Und dere ich meh als dir zdanke ha.

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842843-b3c9ce1e-cf9e-4e5d-bb67-46767884c4aa.jpeg)

**Reply by ThomasG777** [reply]
Über die Felder

Über de Himmel Wolke ziehnd
Über die Felder gaht de Wind
Über die Felder wandered 
Miner Muetter verlorenes Chind

Über die Strasse Blätter wehed,
Über de Bäum Vögel schreied-
Irgendwo über de Berge
Muss mini ferni Heimat sie.

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842856-9277fea3-7676-4f9d-b965-d4739758ff64.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842872-85ba467e-7a2c-45ed-b2cc-934c36b00405.jpeg)

**Reply by ThomasG777** [reply]
Genesig

Lang waret mini Auge müed`
Und vom Rauch der Städt bang verschleichert.
nu bin i schudernd erwacht: Fäscht fiered jede
Baum und jede Garte blüeht.

Wieder wie ich mal als Bueb gshe ha.
gseh ich fröhlich durch die sanfte Wite
Engel ihre wisse Flügel usbreite
Und dAuge Gottes, blau und nah.

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842883-99756a4c-20ab-4b4d-a575-555719c06665.jpeg)

**Reply by ThomasG777** [reply]
Zunachte

Laufüechti Wind schweife,
dNachtvögel ghört mer überem Ried
Mit schwere Flügel streife
Und fern im Dorf es Fischerlied.

Us nie gwesene Zite
Sind trüebi Sage agstimmt
Un dKlage um ewigi Lide;
Wehe dem, der sie i de Nacht vernimmt!

Lass klage, Chind, Lass Rusche!
Ring isch dWelt vom Lide schwer.
mir wennd de Vögel luhsche
Und au dem Lied vom Dorfe her.

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842899-30b1ddaf-fa8a-4d5e-986b-1895eb50f7b5.jpeg)

**Reply by ThomasG777** [reply]
Jugendflucht

Der müedi Summer senkt das Haupt
Und luegt sis falbes Bild im See.
Ich wandle müed und bestaubt
im Schatte vo de Allee.

Durch Papple gaht en zage Wind,
De Himmel hinter mir isch rot,
Und vor mir Abigängscht sind
-Und Dämmerig-und Tod.

Ich wandle müed und bestaubt,
Und hinter mir blibt zögernd stah
dJugend, neigt das schöni Haupt
Und will nöd fürder mit mir gah.

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213842914-6361014d-65fa-4cfa-9b33-b66af06ac12c.jpeg)

**Reply by ThomasG777** [reply]
hemer scho lol

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213843073-275f8f22-ae4b-4bfe-a612-f75b18a82faa.jpeg)

**Reply by ThomasG777** [reply]
hemer scho lol2

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213843080-df5dd1b5-fae0-45ff-8c08-e1417a582a72.jpeg)

**Reply by ThomasG777** [reply]
hemer scho lol3

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213843134-db8bd723-958c-480a-b8fc-e44d9eed50a0.jpeg)

**Reply by ThomasG777** [reply]
hemer scho lol5

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/213843145-219a6dca-6cb7-4335-b3f6-9e31728467c1.jpeg)

**Reply by ThomasG777** [reply]
Einsame Abig 

I de Läärä Fläsche und im Glas
Wankd de Cherze Schimmer;
Es isch Chalt im Zimmer.
Dusse fallt de Rägä weich is Gras.

Wieder legsch di nue zur churzer Rueh
Frürend und trurig nieder.
Morge chunt und Abig wieder
chömet immer wieder,
Aber niemals du.

**Comment by ThomasG777** [comment]
Büecher

Alle Büecher vo  dere Welt
Bringed dir keis Glück,
Doch sie weised dich geheim 
I dich selbscht zrug.

Det isch alles, was du bruchsch,
Sunne ,Stern und Mond,
Denn das Liecht, das du suechsch
I dir selber wohnt.

Wisheit ,die du lang hesch gsucht
I de Büechereie,
Lüchted jetzt us jedem Blatt-
denn nun isch sie dir

**Comment by ThomasG777** [comment]
Vergänglichkeit

Vom Baum vom Lebe fallt
Mir Blatt um Blatt,
O taumelbunti Welt,
Wie machsch du satt,
Wie machsch du satt und müed,
Wie machsch du trunke!
Was hüt no glüeht,
Isch bald versunke.
Bald klirrt de Wind 
Über mis Brune Grab,
Über das chline Chind
Bügt sich dmuetter herab.
Ihre Auge will i wiedergseh,
Ihr Blick isch min Stern,
Alles anderi mag gah und verwehe,
Alles stirbt, alles stirbt gern.

**Comment by ThomasG777** [comment]
Himmel uf Erde

es isch en traum
alles isch da 
es isch min traum
ich nimm en a

ich bin mittedrin
im schönschte wos git
bitte blieb wahr
verlass mich nit

ich folge de zeiche 
ich luege nach vorn
ich wird sie erreiche
am beschte scho morn

alle isch möglich
und no viel meh
glaubed a euch
und ihr werdets gseh



TG2023
